app/assets/javascripts/govuk_publishing_components/components/copyable-content.js

```diff
@@ -0,0 +1,25 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (global, GOVUK) {
+  "use strict"
+
+  var $ = global.jQuery
+
+  GOVUK.Modules.CopyableContent = function () {
+    this.start = function (element) {
+      element.on("click", "button", function (event) {
+        event.preventDefault()
+        copyToClipboard(element.find("input").val())
+      })
+    }
+
+    function copyToClipboard(text) {
+      var $temp = $("<input>");
+      $("body").append($temp);
+      $temp.val(text).select();
+      document.execCommand("copy");
+      $temp.remove();
```

Comment:
I'd imagine you could achieve this with the same amount of lines without having the dependency on jQuery

Reply:
Show me!